hedera-node/src/main/java/com/hedera/services/context/properties/GlobalDynamicProperties.java

```diff
@@ -521,6 +523,8 @@ public boolean isHTSPrecompileCreateEnabled() {
 		return enableHTSPrecompileCreate;
 	}
 
+	public boolean isDirectHTSTokenCall(){return enableDirectHTSTokenCalls;}
```

Comment:
This will be more readable if the method is also called `enableDirectHTSTokenCalls()`